nixos/modules/services/web-apps/lemmy.nix

```diff
@@ -16,6 +16,20 @@ in
 
     enable = mkEnableOption (lib.mdDoc "lemmy a federated alternative to reddit in rust");
 
+    packages.lemmy-server = mkOption {
+      type = types.package;
+      description = lib.mdDoc "lemmy-server package to use.";
+      default = pkgs.lemmy-server;
+      defaultText = literalExpression "pkgs.lemmy-server";
+    };
+
+    packages.lemmy-ui = mkOption {
```

Comment:
Should I:

1. put this here as `services.lemmy.packages.lemmy-ui` like gitlab does it
2. do it like `services.lemmy.ui.package` and rename `services.lemmy.packages.lemmy-server` to `services.lemmy.package`

?

Reply:
(The diff is outdated but the question still stands.)

Reply:
my comment wasn't very clear, I would say go for something like 2
create a `services.lemmy.server.package`